static/app/actions/projectActions.tsx

```diff
@@ -1,27 +1,5 @@
 import {createActions} from 'reflux';
 
-const ProjectActions = createActions([
-  'addTeam',
-  'addTeamError',
-  'addTeamSuccess',
-  'changeSlug',
-  'createSuccess',
-  'loadProjects',
-  'loadStats',
-  'loadStatsError',
-  'loadStatsForProjectSuccess',
-  'loadStatsSuccess',
-  'removeProject',
-  'removeProjectError',
-  'removeProjectSuccess',
-  'removeTeam',
-  'removeTeamError',
-  'removeTeamSuccess',
-  'reset',
-  'setActive',
-  'update',
-  'updateError',
-  'updateSuccess',
-]);
+const ProjectActions = createActions(['createSuccess']);
```

Comment:
This one is difficult to get rid of because there is a `.listen` call _specifically on it_.

I'll figure out how once I get this in

Reply:
The one in `OrganizationContextContainer`? it's weird we're setting up a listener on an `willUnmount` anyway